Public/Src/Cache/ContentStore/Distributed/NuCache/ContentLocationDatabase.cs

```diff
@@ -106,10 +108,17 @@ public static ContentLocationDatabase Create(IClock clock, ContentLocationDataba
             }
         }
 
+        /// <todoc />

```

Comment:
Don't leave <todoc /> on public methods.

Reply:
This is a placeholder, kind of WIP. So yep, we should comment this.

---
In reply to: [292137263](https://github.com/microsoft/BuildXL/pull/403#discussion_r292137263) [](ancestors = 292137263)